core/src/main/java/org/opensearch/sql/expression/function/OpenSearchFunctions.java

```diff
@@ -59,8 +61,8 @@ private static FunctionResolver match_phrase(BuiltinFunctionName matchPhrase) {
     return new RelevanceFunctionResolver(funcName, STRING);
   }
 
-  private static FunctionResolver multi_match() {
-    FunctionName funcName = BuiltinFunctionName.MULTI_MATCH.getName();
+  private static FunctionResolver multi_match(BuiltinFunctionName matchPhrase) {
```

Comment:
Oops I think this was my mistake when I was still working on this, but would you mind changing the variable `matchPhrase` to `multimatch` or something similar? This was probably left over when I was copying how it was implemented in the `match_phrase` function just above this.